docs/posts/introduction-to-ballerina-0.970.md

```diff
@@ -0,0 +1,220 @@
+
+## Textual Syntax
+Ballerina’s textual syntax is largely inspired by C, Java, and Go languages. The key language constructs in Ballerina are Function, Worker and Service. 
+
+## Graphical Syntax
+Ballerina’s graphical syntax resembles a sequence diagram. The control flow within a worker is represented with flow diagram based elements. Graphical representation of a service is represented with network interactions.
+
+Ballerina platform comes with the Composer IDE, which allows you to edit and view Ballerina programs graphically and textually. VS Code plugin can be also used to view Ballerina programs graphically.
+
+# Integration Specialization 
+Ballerina has first class support for services and endpoints. HTTP/HTTP2, WebSockets, WebSub, gRPC, and JMS are some of the available service types. These services are exposed via listener endpoints, which can be secured and monitored. Client endpoints connect to different types of external endpoints and they are inherently resilient. Additionally, commonly used integration message formats, such as XML and JSON, are built-in to the type system of the language. 
+
+In the context of integration specialization, the following are the released features.
+
+## HTTP 1.x/2 
+Ballerina includes HTTP 1 and 2 support with server and client endpoints. 
+
+## WebSockets
+WebSocket client/server endpoints. 
+
+## Resiliency
+Support for network resiliency with Circuit breaker, retry, timeout, load balancing and failover. 
+
+## MIME
+Ballerina provides built-in implementation of the MIME specification.  
+
+## gRPC
+Ballerina based gRPC server/client supports interacting with server and client written either in Ballerina or any other supported languages. 
+
+## Database client endpoints
+Ballerina Database client endpoints allow you to connect to SQL-based relational database systems and perform data definition, data access, and data manipulation operations on the database. These database client endpoints are supported: JDBC for any JDBC supported database, MySQL, H2. 
+
+## Messaging endpoints
+Messaging connectors enable the services and programs written in Ballerina to communicate with messaging backends like Ballerina Message Broker and ActiveMQ. The messaging connector API provides features like transactions, message headers, properties, and different acknowledgement mode support to address the common requirements of a modern integration engineer when integrating with a messaging system.
+
+## WebSub
+Implementation of the WebSub recommendation that facilitates push-based content delivery/notification mechanism between publishers and subscribers.
+
+# Standard Library 
```

Comment:
Can we reformat or restructure so that package names are not used as titles? Perhaps we can add a table or a list.